libs/dh/wholesale/feature-request-calculation/src/lib/dh-wholesale-request-calculation.ts

```diff
@@ -244,23 +244,15 @@ export class DhWholesaleRequestCalculationComponent {
             this.form.controls.energySupplierId.setValue(glnOrEicNumber);
           }
         }),
-        switchMap(({ marketRole }) =>
-          this.form.controls.calculationType.valueChanges.pipe(
-            startWith(this.form.controls.calculationType.value),
-            exists(),
-            map((value) => wholesaleCalculationTypes.includes(value)),
-            tap((isWholesale) => {
-              if (isWholesale && marketRole !== EicFunction.GridAccessProvider) {
-                this.gridAreaIsRequired.set(false);
-                this.form.controls.gridArea.removeValidators(Validators.required);
-              } else {
-                this.gridAreaIsRequired.set(true);
-                this.form.controls.gridArea.setValidators(Validators.required);
-              }
-            }),
-            tap(() => this.form.controls.gridArea.updateValueAndValidity())
-          )
-        ),
+        tap(({ marketRole }) => {
+          if (marketRole !== EicFunction.GridAccessProvider) {
+            this.gridAreaIsRequired.set(false);
+            this.form.controls.gridArea.removeValidators(Validators.required);
+          } else {
+            this.gridAreaIsRequired.set(true);
+            this.form.controls.gridArea.setValidators(Validators.required);
+          }
```

Comment:
question: no need for `updateValueAndValidity` any longer?

Reply:
Doesn't seem like it is needed any more, since changing actors reloads the entire view. Before it was needed since the validators would be updated when selecting different options in the dropdown.

Reply:
My understanding was that `updateValueAndValidity` needs to be called every time a validator is added/removed at run time. The documentation also mentions that. But trying the code locally it seems like everything works fine. Let's go without it and keep it in mind in case some weird behaviour is observed. 

https://angular.dev/api/forms/AbstractControl#addValidators

![image](https://github.com/user-attachments/assets/539b3e37-0ae8-427d-b644-0334fc8762be)